src/js/components/tour/plan.tsx

```diff
@@ -0,0 +1,139 @@
+import i18n from 'i18next';
+import React, { useCallback, useState } from 'react';
+import { Trans } from 'react-i18next';
+import Joyride, {
+  ACTIONS,
+  CallBackProps,
+  STATUS,
+  status as StatusType,
+  Step,
+  StoreHelpers,
+} from 'react-joyride';
+
+import PopoverHeader from '~img/popover-header.png';
+
+interface Props {
+  run: boolean;
+  onClose: () => void;
+}
+
+const PlanTour = ({ run, onClose }: Props) => {
+  const [helpers, setHelpers] = useState<StoreHelpers | null>(null);
+
+  const steps: Step[] = [
+    {
+      target: '.tour-create-epic',
+      title: i18n.t('Create Epics to group Tasks'),
+      content: (
+        <Trans i18nKey="tourCreateEpic">
+          Epics are groups of related Tasks, representing larger changes to the
+          Project. You can invite multiple Collaborators to your Epic and assign
+          different people as Developers and Testers for each Task.
+        </Trans>
+      ),
+      placement: 'right',
+      disableBeacon: true,
+    },
+    {
+      target: '.tour-walkthroughs',
+      title: i18n.t('Review walkthroughs any time'),
+      content: (
+        <Trans i18nKey="tourWalkthroughs">
+          If you would like to review this or any other walkthrough again,
+          simply click the question mark. There is also a self-guided tour with
+          more object and action explanations.
+        </Trans>
+      ),
+      placement: 'right',
+      disableBeacon: true,
+    },
+  ];
+
+  const handleCallback = useCallback(
+    (data: CallBackProps) => {
+      const { action, status } = data;
+      const finished: StatusType[keyof StatusType][] = [
+        STATUS.FINISHED,
+        STATUS.SKIPPED,
+      ];
+      if (finished.includes(status) || action === ACTIONS.CLOSE) {
+        /* istanbul ignore else */
+        if (helpers) {
+          helpers.close();
+        }
+        onClose();
+      }
+    },
+    [onClose, helpers],
+  );
+
+  return (
+    <Joyride
+      steps={steps}
+      run={run}
+      locale={{
+        back: i18n.t('Back'),
+        close: i18n.t('Close'),
+        last: i18n.t('Close'),
+        next: i18n.t('Next'),
+      }}
+      continuous
+      showProgress
+      disableScrolling
+      getHelpers={setHelpers}
+      callback={handleCallback}
+      styles={{
+        options: {
```

Comment:
**suggestion**: define these styles outside of this file for reuse other tours.

(Probably not a blocker here, but we may want this abstracted in future tour stories)